### DISCUSSION TITLE
[Feature Request] Clickhouse support

### DISCUSSION THREAD

**Discussion by DaniFoldi** [sqlpage/SQLPage#521] [Ideas]
Hi :wave:,

Most of that I could test SQLpage on lives in Clickhouse databases. Would you be open to considering adding support for native clickhouse?

**Comment by lovasoa** [comment]
Hello! 

 Support for more databases is on the roadmap... Would you be interested in helping with the implementation? Ideally I would like to add support for odbc to https://github.com/lovasoa/sqlx-oldapi

 In the meantime, you should be able to access your clickhouse data by setting up a small postgres instance and using https://github.com/ildus/clickhouse_fdw

  Related: https://github.com/lovasoa/SQLpage/discussions/226

**Reply by DaniFoldi** [reply]
Hi,
Thanks for the quick response!

Of course, I'll give the fdw a try, and if I have some time, I'd be happy to help (if nothing else, with testing)!
